pkg/notes/notes.go

```diff
@@ -252,6 +252,7 @@ func NoteTextFromString(s string) (string, error) {
 		note := strings.ReplaceAll(result["note"], "#", "&#35;")
 		note = strings.ReplaceAll(note, "\r", "")
 		note = stripActionRequired(note)
+		note = strings.TrimSpace(note)
```

Comment:
Please remove this line, its a duplicate with the one below.

Reply:
@paulbouwer -- yep. Once this duplicate line is pulled, this looks ready to merge. :)

Reply:
@justaugustus - done.